pymc3/distributions/discrete.py

```diff
@@ -792,6 +792,115 @@ def _repr_latex_(self, name=None, dist=None):
         return r'${} \sim \text{{Geometric}}(\mathit{{p}}={})$'.format(name,
                                                 get_variable_name(p))
 
+class HyperGeometric(Discrete):
+    R"""
+    HyperGeometric log-likelihood.
+
+    The probability of x successes in a sequence of n Bernoulli
+    trials (That is, sample size = n) - where the population 
+	size is N, containing a total of k successful individuals.
+	The process is carried out without replacement.
+	
+
+    The pmf of this distribution is
+
+    .. math:: f(x \mid N, n, k) = \frac{\binom{k}{x} \binom{N-k}{N-x}}{\binom{N}{n}} 
+
+    .. plot::
+
+		import matplotlib.pyplot as plt
+		import numpy as np
+		import scipy.stats as st
+		plt.style.use('seaborn-darkgrid')
+		x = np.arange(1, 15)
+		N = 50
+		k = 10
+		for n in [20, 25]:
+		    pmf = st.hypergeom.pmf(x, N, k, n)
+		    plt.plot(x, pmf, '-o', label='n = {}'.format(n))
+		plt.plot(x, pmf, '-o', label='N = {}'.format(N))
+		plt.plot(x, pmf, '-o', label='k = {}'.format(k))
+		plt.xlabel('x', fontsize=12)
+		plt.ylabel('f(x)', fontsize=12)
+		plt.legend(loc=1)
+		plt.show()
+
+    ========  =============================
+    Support   :math:`x \in \mathbb{N}_{>0}`
+    Mean      :math:`\dfrac{n.k}{N}`
+    Variance  :math:`\dfrac{(N-n).n.k.(N-k)}{(N-1).N^2}`
+    ========  =============================
+
+    Parameters
+    ----------
+    N : integer
+        Total size of the population
+    n : integer
+    	Number of samples drawn from the population
+    k : integer
+    	Number of successful individuals in the population
+    """
+
+    def __init__(self, N,  k, n, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.k = k = tt.as_tensor_variable(floatX(k))
```

Comment:
These need to be `intX` instead of `floatX`

Reply:
Done, thanks!